content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,218 @@
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Navigation]
+
+Docs: provide a link to the updated docs	
+
+### Access code context while staying within the developer workflow 
+Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, are now part of the core functionality. These enhancements provide developers code context while staying in the flow within their preferred tools. Additionally, the current Sourcegraph extensions framework will be deprecated so that we can make way for a new model that allows a deeper integration throughout our entire code intelligence platform. Please note, these changes do not impact our IDE extensions. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Integrations]
+
+Docs: provide a link to the updated docs	
+##### 🏗️ High-leverage ways to improve your entire codebase
+
+### Make changes across all of your codebases at enterprise scale with server-side Batch Changes (beta)
+Batch Changes can now run across thousands of repositories with a better development experience. Previously, you had to run src-cli locally to create batch changes. That could take an impractical amount of time or be brittle for large or resource-intensive batch changes. By running Batch Changes server-side you can:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+[Batch Changes]
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+- New packaging updates
+
+##### ☁️ Enterprise Cloud dedicated(/managed?) instances
+
+- [Secure, easy, and scalable Sourcegraph Cloud dedicated(/managed?) instances for the enterprise](/blog/enterprise-cloud)
+
+Headline: [Do X] with [Z]
+Description: 2 - 4 sentences on the feature, functionality and benefits
+Media: provide a screenshot, gif, or video to help support the copy
+Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
+Docs: provide a link to the updated docs
```

Comment:
@iskyOS seciont to be competed for Cloud

Reply:
Since there is only one sub-section under this "Cloud" section, do we want a second headline?